### DISCUSSION TITLE
AIO Interface only works some of the time

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#2324] [Questions] [answered]
### Steps to reproduce
1. Deploy Nextcloud AIO through Linode Marketplace
2. Click "Open Nextcloud AIO Interface" to check for updates
3. Sometimes the AIO interface appears, sometimes it doesn't. I have no idea why this is.

### Expected behavior
For the AIO interface to always appear.

### Actual behavior
Just says "The page can't connect".

### Host OS
Ubuntu 22.04

#### Nextcloud AIO version
I currently cannot access the interface. I did however access it a few weeks ago and performed an update without issue.

#### Current channel 
Again, the interface is unavailable.

#### Other valuable info
I have not moved the server/changed IP/domain or done anything else.

**What I want to know**
How can I update the docker images with the command line, so I don't have to rely on the AIO web interface? What exact steps would be involved in doing that?

**Comment by szaimen** [comment]
Moving to discussions since I cannot reproduce this

**Comment by szaimen** [comment]
@stianmandarin is it possible that the server has a very high load when you are not able to log in? Can you check this when this happens e.g. with htop? 

Also it would be usefuly to get your mastercontainer logs when this happens. You can get them with `sudo docker logs nextcloud-aio-mastercontainer` then.

**Reply by ghost** [reply, marked as answer]
Very low memory and CPU usage. However, after trying various things at random, I found that disabling crowdsec did the trick. So I'm thinking that something on the crowdsec side must've changed recently to prevent AIO from working properly. I suppose going forward disabling crowdsec, updating via AIO interface, then enabling crowdsec again is what I must do.
